modules/apps/transfer/module.go

```diff
@@ -126,6 +126,10 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	if err := cfg.RegisterMigration(types.ModuleName, 3, m.MigrateParams); err != nil {
 		panic(fmt.Sprintf("failed to migrate transfer app version 3 to 4 (self-managed params migration): %v", err))
 	}
+
+	if err := cfg.RegisterMigration(types.ModuleName, 4, m.MigrateDenomMetadata); err != nil {
+		panic(fmt.Sprintf("failed to migrate transfer app from version 4 to 5 (set denom metadata migration): %v", err))
+	}
```

Comment:
We could wrap both params migration and denom metadata in an anonymous func here, but having two separate also works. 

Happy to keep as is if everyone is fine with bumping consensus version twice in the one release version.

Reply:
I'm fine keeping as is. One benefit is that it makes it quite clear which migration is doing what

Reply:
Agreed! Will resolve this comment :)